### ISSUE TITLE
Stale bot is a bit aggressive 

### ISSUE THREAD

**Issue by 3flex** [detekt/detekt#5520]
## Expected Behavior
Issues shouldn't always be auto closed just because they haven't been interacted with in a while.

## Observed Behavior
Issues are marked for closure after 90 days, then closed a week later. Some valid issues are being closed, and it creates noise for anyone who left a comment or subscribed to updates.

## Context
I think stale bot should only auto close:
- PRs if there is no recent activity, possibly with a shorter threshold
- Issues, but only those tagged with "waiting for feedback" that have been open more than a certain time. 30 days might be more appropriate here.

## Your Environment

* Version of detekt used:
* Version of Gradle used (if applicable):
* Gradle scan link (add `--scan` option when running the gradle task):
* Operating System and version:
* Link to your project (if it's a public repository):

**Comment by BraisGabin** [comment]
I don't think it is too aggressive. 3 months without an update is usually means that the issue is not actionable. I would also like to know how much issues was closed but stale bot. I think they were not too much.

I feel like stale bot force us to take a decision over those type of issues? Can we actionate it? Is clear enoght for any one to pick it up? We end up taking the decission of adding a tag to avoid the stealing (for example help wanting) or taking the decission to close the issue ourself before the 7 days period.

And even if stale bot ends up closing an issue there is nothing preventing us to reopen the issue of we really think that it is worth it.

**Comment by BraisGabin** [comment]
> Issues, but only those tagged with "waiting for feedback" that have been open more than a certain time. 30 days might be more appropriate here.

I also though about this. The waiting for feedback issues should stale faster.

**Comment by cortinico** [comment]
> I don't think it is too aggressive.

Same. I actually believe the value it brings is net positive. I agree that some valuable issues might be closed, but so far it also helped us bring attention and solve issues that we forgot to follow up with.

> The waiting for feedback issues should stale faster.

Agree. I've sent this one here:
- https://github.com/detekt/detekt/pull/5543
